src/main/java/uk/gov/hmcts/reform/idam/health/vault/AccessTokenKeyVaultCredential.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.idam.health.vault;
```

Comment:
Would be nice to move the two credential classes to a package structure that would also work for the JCE repo